controllers/collection.lua

```diff
@@ -87,38 +74,91 @@ CollectionController.GET.user_collections = function (self)
     end
 end
 
-CollectionController.GET.collection = function (self)
+CollectionController.GET.collections = function (self)
     -- GET /users/:username/collections/:collection_slug
     -- Description: Get info about a collection.
     -- Parameters:  username, collection_slug, ...
 
-    return -- TODO
-    -- local collection = assert_collection_exists(self)
-    -- local project_count = collection:count_projects
-    -- collection.projects_count = project_count
-    -- return jsonResponse(collection)
+    local collection = assert_collection_exists(self)
+    local project_count = collection:count_projects()
+    collection.projects_count = project_count
+    -- TODO: figure out a better way around this.
+    collection.shared_at = tostring(collection.shared_at)
```

Comment:
Ok so so,etching is weird.  Maybe I just noticed this but without doing this those fields don’t end up in the final response. But tostring isn’t quite the right method either.